### ISSUE TITLE
Feature Request: Percetange-based value and integrated dynamic line chart history

### ISSUE THREAD

**Issue by LostAccount** [crystalidea/macs-fan-control#181]
Hello

When choosing the Custom controls, a % value could be useful. For example, a percentage-based value of say 10% above what is Apple recommended might be a welcome method of increasing fan speed.

If the app were able to display a time 'line chart' showing fan RPM and CPU/GPU speed (throttling) over a given time frame one might be able to extrapolate how much to increase fan speed by to find a sweet spot that might be different for each user. The time 'line chart' could even be dynamic (react in real time to an adjustment) so that if a user adjusts a percentage value we could see how the history in the line chart might have looked had a different RPM % or any other value been set differently providing insight on what might work best.

To summarize or in an effort to clarify my last paragraph, if a line chart showing three lines corresponding to time history of heat, CPU/GPU and fan speed could be designed to react to a fan speed slider to get a sense of how history might have been different had the fan speed variable been set differently.

Maybe it can assume a look like this?
![Screen Shot 2019-06-14 at 11 42 30](https://user-images.githubusercontent.com/2214319/59500363-90ae6700-8e99-11e9-9404-950df83dc8f5.png)

NOTE: 

I read your guidelines on introducing a feature request but introducing screenshots will not introduce any value here.
